registered/validate/validators.py

```diff
@@ -516,6 +517,27 @@ def validate_all_runs_have_blocks(rating):
         )
 
 
+def validate_run_id_format(rating):
+    """
+    Validate each Run ID in each Piece in the CRW file is in the format of 
+    "3 digits" + "-" + "4 digits" ex 123-0001
+    """
+
+    for piece in rating["crw"]:
+        if not isinstance(piece, parser.Piece):
+            continue
+
+        if bool(re.fullmatch(r"\d{3}-\d{4}", piece.run_id)):
```

Comment:
nitpick: you don't need the `bool` here, since that's how the `if` statement works anyways